mod/community.php

```diff
@@ -186,23 +206,40 @@ function community_content(App $a, $update = 0)
 	]);
 }
 
-function community_getitems($start, $itemspage, $content)
+function community_getitems($start, $itemspage, $content, $accounttype)
```

Comment:
Please add a default value for $accounttype if a value isn't required like this: `, $accounttype = null)`

Reply:
This function is only used at a single place where the variable is set. So defining a default value shouldn't be needed. I only use default values when a function is called from different places.

Reply:
Not necessarily. No matter how it is used right now, a function declaration should be relevant to any uses you can possibly think of. Since there is a provision for a null value for the `$accounttype` parameter in the function body, there shouldn't be a need to have a potential future call like this: `community_getitems($start, $itemspage, $content, null)`.

Better set the parameter as optional as it is meant to be.